generated/nitclk/session.py

```diff
@@ -0,0 +1,1156 @@
+    originally generates the trigger.
+    '''
+    start_trigger_master_session = _attributes.AttributeViInt32SessionReference(3)
+    '''Type: nimi-python Session class, nitclk.SessionReference, NI-TClk Session Number
+
+    Specifies the start trigger master session.
+    For external triggers, the session that originally receives the trigger.  For None (no trigger configured) or software triggers, the session that  originally generates the trigger.
+    '''
+    sync_pulse_clock_source = _attributes.AttributeViString(10)
+    '''Type: str
+
+    Specifies the Sync Pulse Clock source. This property is typically used to  synchronize PCI devices when you want to control RTSI 7 yourself. Make  sure that a 10 MHz clock is driven onto RTSI 7.
+    Values
+    PCI Devices -  'RTSI_7' and  'None'
+    PXI Devices -  'PXI_CLK10' and  'None'
+    Default Value -  'None' directs synchronize to create the necessary routes. For  PCI, one of the synchronized devices drives a 10 MHz clock on RTSI 7  unless that line is already being driven.
+    '''
+    sync_pulse_sender_sync_pulse_source = _attributes.AttributeViString(13)
+    '''Type: str
+
+    Specifies the external sync pulse source for the Sync Pulse Sender.  You can use this source to synchronize  the Sync Pulse Sender with an external non-TClk source.
+    Values
+    Empty string. Empty string is a valid value, indicating that the signal is  not exported.
+    PXI Devices -  'PXI_Trig0' through  'PXI_Trig7' and device-specific settings
+    PCI Devices -  'RTSI_0' through  'RTSI_7' and device-specific settings
+    Examples of Device-Specific Settings
+    - NI PXI-5122 supports  'PFI0' and  'PFI1'
+    - NI PXI-5421 supports  'PFI0',  'PFI1',  'PFI4', and  'PFI5'
+    - NI PXI-6551/6552 supports  'PFI0',  'PFI1',  'PFI2', and  'PFI3'
+    Default Value is empty string
+    '''
+    sync_pulse_source = _attributes.AttributeViString(1)
+    '''Type: str
+
+    Specifies the Sync Pulse source. This property is most often used when  synchronizing a multichassis system.
+    Values
+    Empty string
+    PXI Devices -  'PXI_Trig0' through  'PXI_Trig7' and device-specific settings
+    PCI Devices -  'RTSI_0' through  'RTSI_7' and device-specific settings
+    Examples of Device-Specific Settings
+    - NI PXI-5122 supports  'PFI0' and  'PFI1'
+    - NI PXI-5421 supports  'PFI0',  'PFI1',  'PFI2', and  'PFI3'
+    - NI PXI-6551/6552 supports  'PFI0',  'PFI1',  'PFI2', and  'PFI3'
+    Default Value - Empty string. This default value directs  synchronize to set this property when all the synchronized devices  are in one PXI chassis. To synchronize a multichassis system, you must set  this property before calling synchronize.
+    '''
+    tclk_actual_period = _attributes.AttributeViReal64(8)
+    '''Type: float
+
+    Indicates the computed TClk period that will be used during the acquisition.
+    '''
+
+    def __init__(self, session, repeated_capability_list='', encoding='windows-1251'):
```

Comment:
I thought we were not going to support `repeated_capability_list`?
Should `session` be `session_handle` or `vi`? (is it the number?)

Reply:
`session` came from the initial metadata (CVI) but we can make it what ever we want.

`repeated_capabilities_list` removed. Gives you more files to review.